bower.json

```diff
@@ -83,8 +86,11 @@
     "knockout": {
       "js": "dist/knockout.debug.js"
     },
-    "lru-ts": {
-      "js": "dist/global/lru-ts.js"
+    "lodash": {
+      "js/": "dist/*.js"
```

Comment:
Ok. I am also not happy about `lodash`. Let's face this topic together on Monday.